x-pack/plugins/security_solution/common/entity_analytics/risk_engine/privileges.ts

```diff
@@ -20,18 +22,19 @@ export interface MissingPrivileges {
 }
 
 export const getMissingIndexPrivileges = (
-  privileges: EntityAnalyticsPrivileges['privileges']['elasticsearch']['index']
+  privileges: EntityAnalyticsPrivileges['privileges']['elasticsearch']['index'],
+  required?: NonEmptyArray<RiskEngineIndexPrivilege>
```

Comment:
We could default `required` to `defaultRequiredPrivileges`

Reply:
we can't default to that specific variable because it's deconstructed from the `RISK_ENGINE_REQUIRED_ES_INDEX_PRIVILEGES` object

We could do something like `required = RISK_ENGINE_REQUIRED_ES_INDEX_PRIVILEGES[RISK_SCORE_INDEX_PATTERN]` but that only works for that specific index. Granted, right now it's the only index we have, but I can imagine adding more in the future and then being confused as to why we're getting defaults only from the RISK_SCORE one.

Probably the best way is to just do a literal default like `required: NonEmptyArray<RiskEngineIndexPrivilege> = ['read', 'write']` ?